src/controllers/accounts/AccountModel.cpp

```diff
@@ -26,7 +26,7 @@ void AccountModel::getRowFromItem(const std::shared_ptr<Account> &item,
 }
 
 int AccountModel::beforeInsert(const std::shared_ptr<Account> &item,
-                               std::vector<QStandardItem *> &row,
+                               std::vector<QStandardItem *> &,
```

Comment:
The argument missing is way more confusing here. Try changing it to `std::vector<QStandardItem *> &, /* row */,`

Reply:
Done